lib/LaTeXML/Package/graphics.sty.ltxml

```diff
@@ -34,6 +35,27 @@ DefParameterType('GraphixDimension', sub {
       $gullet->readDimension; } },
   optional => 1);
 
+# For trim, viewport: a sequence of 4 dimensions
+# These must be space separated, but apparently trailing commas are ignored.
+DefParameterType('GraphixDimensions', sub {
+    my ($gullet) = @_;
+    $gullet->skipSpaces;
+    my @dims = ();
+    while (scalar(@dims) < 4) {
+      if (scalar(@dims)) {
+        my $t = $gullet->readToken;
+        if ($t && !$t->equals(T_OTHER(','))) {
+          $gullet->unread($t); } }
+      my $s = $gullet->readOptionalSigns();
+      if (defined(my $d = $gullet->readRegisterValue('Dimension', $s, 1))) {
+        push(@dims, $d); }
+      elsif (defined($d = $gullet->readFactor())) {
+        my $unit = $gullet->readUnit() || $STATE->convertUnit('bp');
+        push(@dims, Dimension(fixpoint($s * $d, $unit))); }
+      else { last; } }
+    return LaTeXML::Core::Array->new(values => [@dims], separator => T_SPACE); },
```

Comment:
minor: I think `[@dims]` does a copy, while `\@dims` would pass the current ones by reference.

Reply:
ok